src/common/select-objects.js

```diff
@@ -68,6 +69,26 @@ export class GenericSelect extends Component {
     }
   }
 
+  _getValue (xoObjectsById = this.state.xoObjectsById, props = this.props) {
+    const { value } = this.state
+
+    if (props.multi) {
+      // Returns the values of the selected objects
+      // if they are contained in xoObjectsById.
+      return reduce(value, (acc, value) => {
+        const o = xoObjectsById[value.value || value]
+
+        if (o) {
+          acc.push(o)
+        }
+
+        return acc
+      }, [])
```

Comment:
You can use [`utils/mapPlus`](https://github.com/vatesfr/xo-web/blob/v5.x/src/common/utils.js#L217-L222) for this usage.